README.md

```diff
@@ -10,7 +10,22 @@ Our stretch goals include adding collaborators to any bucket list, viewing popul
 
 ## Project task requirements
 
-*placeholder*
+### Minimal requirements (will definitely complete)
+* User should be able to create, delete, edit buckets of travel destinations of interest
+* User should be able to add, remove, move, destinations within a bucket
+* All user information should persist in a database and retrieved upon site load
+### Standard requirements (will most likely complete)
+* Support for multiple users
+* Support filtering when viewing map
```

Comment:
Can anyone remind me what was this about? Sorry I forgot :P

Reply:
It's filtering by categories and stuff

Reply:
Ahh right, filtering by destination categories (food, accommodation, etc)? Maybe we can add that in to be more precise.

Reply:
lol sure